docs/world maintainer.md

```diff
@@ -26,8 +26,17 @@ Unless these are shared between multiple people, we expect the following from ea
 ### Adding a World
 
 When we merge your world into the core Archipelago repository, you automatically become world maintainer unless you
-nominate someone else (i.e. there are multiple devs). You can define who is allowed to approve changes to your world
-in the [CODEOWNERS](/docs/CODEOWNERS) document.
+nominate someone else (i.e. there are multiple devs).
+
+### Being added as a maintainer to an existing implementation
+
+At any point, a world maintainer can approve the addition of another maintainer to their world.  
+In order to do this, either an existing maintainer or the new maintainer should open a PR updating the
+[CODEOWNERS](/docs/CODEOWNERS) file.  
+This change must be approved by all existing maintainers of the affected world, the new maintainer candidate, and
```

Comment:
I don't know if you'd want to account for this, but what if somebody is multiple of these things? Does a new Witness or ALttP or SoE etc. maintainer only require one approval?

Reply:
Good point - I think someone opening a PR to be added to Witness would only require my approval, but we could make it such that it requires a different core maintainer as well

Reply:
Interesting. I think it should probably not be necessary because the trust boundary is the same if V approves it for witness or if V approves it for another game.